patroni/ctl.py

```diff
@@ -1181,14 +1182,15 @@ def reinit(cluster_name: str, group: Optional[int], member_names: List[str], for
     :param group: filter which Citus group we should reinit members. Refer to the module note for more details.
     :param member_names: name of the members that should be reinitialized.
     :param force: perform the restart without asking for confirmations.
+    :param from-leader: perform the reinit to sync wal from the leader node.
```

Comment:
Function param name is `from_leader`, not `from-leader`, and the comment is also wrong.